### PR TITLE
fix unbound variable

### PR DESCRIPTION
**Pull request by yunjiang29**
Fix error:
```
/bin/bash: line 62: !v_current_version: unbound variable
```

### CODE REVIEW

**Comment by yunjiang29** [comment]
it works now https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/34601/rehearse-34601-periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-disconnected-private-sno-baselinecaps-none-p2-f7/1600348852437651456/artifacts/aws-ipi-disconnected-private-sno-baselinecaps-none-p2-f7/ipi-conf-manual-creds-remove-unnecessary-creds/build-log.txt

**Comment by yunjiang29** [comment]
@jianlinliu PTAL.

File: ci-operator/step-registry/ipi/conf/manual-creds/remove-unnecessary-creds/ipi-conf-manual-creds-remove-unnecessary-creds-commands.sh [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 set -o errexit
 set -o pipefail
+set +o nounset
```
Comment (jianlinliu) [inline ci-operator/step-registry/ipi/conf/manual-creds/remove-unnecessary-creds/ipi-conf-manual-creds-remove-unnecessary-creds-commands.sh:5]: How about this:
```
set -o nounset
<--snip-->
if [[ ${!v_current_version:-} == "" ]]; then
```

**Pull request merged**